assets/app/scripts/directives/nav.js

```diff
@@ -53,11 +53,16 @@ angular.module('openshiftConsole')
 
           sortedProjects = $filter('orderByDisplayName')(projects);
 
-          options = _.map(sortedProjects, function(item) {
+          options = _.map(sortedProjects, function(item, index) {
```

Comment:
To avoid duplicate code in the directive and in the controller. I'm thinking to build a service for that. What do you think about this proposal ?